app/views/claims/pages/feedback.html.erb

```diff
@@ -0,0 +1,9 @@
+<% content_for(:page_title) { t(".page_title") } %>
+
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l"><%= t(".page_title") %></h1>
+    </div>
+  </div>
+</div>
```

Comment:
Do we know for sure that a standalone feedback page is required?

I'm used to seeing the "feedback" link open an external survey/feedback form. But I expect @simonwhatley will have an opinion on this.

Reply:
The last we spoke about this topic, I think we were going to take the simplest route, similar to [apply's feedback page](https://www.apply-for-teacher-training.service.gov.uk/candidate/make-a-complaint).

Ref: https://ukgovernmentdfe.slack.com/archives/C06962474AC/p1706277019469919?thread_ts=1706274737.868929&cid=C06962474AC

Reply:
It's also very easy to drop this if we don't need it.